flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/exceptions/DatabaseNotExistException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.catalog.exceptions;
+
+/**
+ * Exception for trying to operate on a database that doesn't exist.
+ *
+ */
+public class DatabaseNotExistException extends RuntimeException {
```

Comment:
nit: `DatabaseNotFoundException`? `NoSuchDatabaseException`? My personal feeling, but `DatabaseNotExistException` sounds strange to me.

Reply:
Yeah. I'm not particular on the naming but those names actually came from Flink exceptions.scala. I only migrated them to its java counterparts. Let me know if you think it's time to correct the names.

Reply:
Got it. I would be in favor of renaming them then. We are completely reworking the API, so I think we can fix the naming as well.

Reply:
I see. If you don't mind, I'd open a new JIRA on renaming the classes since this PR is almost ready and it didn't introduce the unfavorable names. Thanks.

Reply:
I would also prefer fixing the names once and forall. Especially because we add `@PublicEvolving` annotations now.

Reply:
Sounds good. I create a new JIRA for fixing the names.

Reply:
Created https://issues.apache.org/jira/browse/FLINK-12255 to track the renaming.